module-sys/SystemManager/SystemManagerCommon.cpp

```diff
@@ -513,17 +505,14 @@ namespace sys
                 case Code::CloseSystem:
                     CloseSystemHandler(data->closeReason);
                     break;
-                case Code::Update:
-                    UpdateSystemHandler();
-                    break;
                 case Code::Restore:
                     RestoreSystemHandler();
                     break;
                 case Code::Reboot:
-                    RebootHandler(State::Reboot);
+                    RebootHandler();
                     break;
                 case Code::RebootToUpdate:
-                    RebootHandler(State::RebootToUpdate, data->updateReason);
+                    RebootToUpdateHandler(data->updateReason);
```

Comment:
please double check if we don't double reverse the list of services somewhere now :) tbh I can't see any sane way when it could harm us, though it would be nice to double-check just in case our system might surprise us.

Reply:
Confirmed. Done only once